src/prospectScraper/Program.cs

```diff
@@ -39,720 +33,27 @@ static void Main(string[] args)
-            var node3 = childNodes[3]; //team (and team image)?
-            var teamCity = node3.ChildNodes[0].InnerText
-                                    .Replace("\r", "")
-                                    .Replace("\n", "")
-                                    .Replace("\t", "")
-                                    .TrimEnd();
-            var node5 = childNodes[5]; //Has Child Nodes - Player, School, Position, Reach/Value
-            string playerName = node5.ChildNodes[1].InnerText
-                                    .Replace("\r", "")
-                                    .Replace("\n", "")
-                                    .Replace("\t", "")
-                                    .TrimEnd();
-            string playerSchoolBeforeChecking = node5.ChildNodes[3].InnerText
-                                    .Replace("\r", "")
-                                    .Replace("\n", "")
-                                    .Replace("\t", "")
-                                    .TrimEnd(); // this may have a space afterwards.
-            string playerSchool = CheckSchool(playerSchoolBeforeChecking);
-            string playerPosition = node5.ChildNodes[5].InnerText
-                                    .Replace("\r", "")
-                                    .Replace("\n", "")
-                                    .Replace("\t", "")
-                                    .Replace(" ", "");
-            string reachValue = node5.ChildNodes[9].InnerText
-                                    .Replace("\r", "")
-                                    .Replace("\n", "")
-                                    .Replace("\t", "")
-                                    .Replace(" ", "");
-
-
-            MockDraftPick mdp = new MockDraftPick(pickNumber, teamCity, playerName, playerSchool, playerPosition, reachValue, pickDate);
-            File.AppendAllText($"logs{Path.DirectorySeparatorChar}Prospects.log", "Mock Draft Round: " + mdp.round + Environment.NewLine);
-            File.AppendAllText($"logs{Path.DirectorySeparatorChar}Prospects.log", "Pick Number: " + mdp.pickNumber + Environment.NewLine);
-            File.AppendAllText($"logs{Path.DirectorySeparatorChar}Prospects.log", "Player: " + mdp.playerName + Environment.NewLine);
-            return mdp;
-        }
-
-        public static DateTime ChangeDateStringToDateTime(string scrapedDate, bool parseDate)
-        {
-            //Change date to proper date. The original format should be like this:
-            //" May 21, 2019 2:00 AM EST"
-            bool parseWorks = DateTime.TryParse(scrapedDate, out DateTime parsedDate);
-            string dateInNiceFormat =  String.Empty;
-
-            if (parseDate && parseWorks)
-            {
-                return parsedDate;
-            }
-            else
-            {
-                return DateTime.Now;
-            }
-        }
-        public static string FormatScrapedDate(DateTime scrapedDate)
```

Comment:
I see that this method has been replaced with an inline `.ToString("yyyy-MM-dd");`  I'm cool with that.  Mostly I was trying to salvage the method that had been previously created, and it isn't really necessary.

Reply:
I'll add more refactoring after we get this one merged in and fixed up that should make these classes smaller and more readable. 😄